opennms-doc/guide-admin/src/asciidoc/text/alarm-correlation/situation-feedback.adoc

```diff
@@ -0,0 +1,25 @@
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+[[ga-situation-feedback]]
+
+=== Situation Feedback
+
+==== Introduction
+
+Situation Feedback allows operators to provide real time feedback on Alarm Correlation.
+
+==== Installation
+
+Situation Feedback needs to be enabled by installing the feature from the Karaf shell:
+
+`feature:install opennms-situation-feedback`
```

Comment:
Can we add a note on how to make the feature install persistent? See the bottom of http://docs.opennms.org/opennms/releases/22.0.4/guide-admin/guide-admin.html#_enabling_the_kafka_producer